### ISSUE TITLE
Tracking PyTorch 1.9.0 additions

### ISSUE THREAD

**Issue by NiklasGustafsson** [dotnet/TorchSharp#277]

**Comment by NiklasGustafsson** [comment]
New namespaces:

torch.special

New functions:

torch.linalg.multi_dot
torch.linalg.lstqs
torch.linalg.vector_norm
torch.linalg.matrix_norm
torch.linalg.matrix_power
torch.linalg.det
torch.linalg.eig
torch.linalg.eigvals
torch.linalg.svdvals
torch.linalg.cholesky_ex
torch.linalg.inv_ex

torch.special.entr
torch.special.xlog1py
torch.special.i0e
torch.special.erfc
torch.special.erfinv
torch.special.expm1
torch.special.exp2
torch.special.sigmoid
torch.special.logit

torch.gradient
torch.{hsplit,vsplit,dsplit}
torch.positive
torch.frexp
torch.take_along_dim
torch.assert_async

torch.{cfloat, cdouble}


New arguments:

torch.clamp: Added support for tensor inputs for min and max arguments (#52695, #56367)
torch.index_add: 'alpha'
torch.quantile: 'interpolation'
torch.{std, var, std_mean, var_mean}: 'correction'
torch.{logit, rad2deg, deg2rad, polygamma}: added integer support
sort: 'stable'
nn.Conv{123}d: 'padding=same'
nn.EmbeddingBag: 'padding_idx'
MaxPoo2d: channels last support

New device:

'meta' -- all operations return only shape information

New modules:

torch.nn.LazyBatchNorm{123}d
torch.nn.HuberLoss
torch.nn.Mish
